cmd/kueue/main.go

```diff
@@ -162,6 +162,14 @@ func main() {
 		kubeConfig.UserAgent = useragent.Default()
 	}
 
+	if cfg.Metrics.LocalQueueMetrics != nil {
```

Comment:
Since this feature is not GA we should have this guarded by a feature gate also.